src/index.html

```diff
@@ -1,14 +1,49 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Stop watch</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Stop watch</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Stop watch</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+
+  <body>
+    <div class="stopwatch">
+      <div class="stopwatch__segment">
+        <div class="stopwatch__color"></div>
+        <div class="stopwatch__transparent"></div>
+        <div class="stopwatch__color"></div>
+      </div>
+      <div class="stopwatch__segment">
```

Comment:
Add empty lines between multilines siblings.

```suggestion
      </div>
      
      <div class="stopwatch__segment">
```